src/markdown_parser.py

```diff
@@ -31,6 +31,9 @@ def inline_html(self, html) -> str:
     def block_html(self, html) -> str:
         return escape(html)
 
+    def linebreak(self) -> str:
```

Comment:
Looks like this is unused. Did you mean to call it in line 358 above?

Reply:
This overrides the default from [mistune](https://github.com/lepture/mistune/blob/46a65a978b005651dcc54a340e44aef71f9f7e95/mistune/renderers.py#L163) and is used internally by that parsing library